index.html

```diff
@@ -26,7 +26,7 @@ <h3 class="logo pointer">TESFAYE.T</h3>
 
     <main>
       <section>
-        <h1>I'm Tesfaye <br>Glad to see you!</h1>
+        <h1>I'm Tesfaye <br>Glad to see<br> you!</h1>
```

Comment:
- [ ] Kindly make sure that whenever this navlinks are clicked, the user is automatically bring to the corresponding section :+1: 
![Screenshot from 2023-01-26 15-46-27](https://user-images.githubusercontent.com/80458199/214852157-a339f1ce-f9b8-4a41-897f-f59968eaf0b0.png)

### Example
```bash
<a href="#section-id">Contact</a>
```